wikipedia_cleanup/data_processing.py

```diff
@@ -131,10 +131,13 @@ def get_data(
     where each all files are sorted after
     (page_id, infobox_key, property_name, value_valid_from)
     """
+    if not Path(input_path).is_dir():
+        raise AttributeError("Expected to get a valid input path.")
     if filters is None:
         filters = []
     files = [x for x in Path(input_path).rglob("*.output.json") if x.is_file()]
     files.extend([x for x in Path(input_path).rglob("*.pickle") if x.is_file()])
+    files = sorted(files)
```

Comment:
Clarification: This relies on the implicit knowledge that the files are named
```python
f"{enwiki-20190901-pages-meta-history{i}.xml-p{a}p{b}.output.json.7z}"
```
right? We should document this (I had to look up the naming schema just for this comment anyways).

```suggestion
    files = sorted(files)
     # files are named f"{enwiki-20190901-pages-meta-history{i}.xml-p{a}p{b}.output.json.7z}"
    # where i is some (ascending?) number, and a-b is the range of page ids contained
```

Reply:
They need to end with `.output.json` or with `.pickle` if thats what you mean and a comment on the input types might useful at the top of the method. The `sorted` just sorts the files in any ordering that does not interest us apart from the fact that its deterministic